robottests/targethosts.py

```diff
@@ -0,0 +1,23 @@
+"""Example Robot framework variable file for robottests.
+
+This is an example how to create
```

Comment:
This comment is not correct. This should explain the crl.remotescript usage for this variable file. Preferably README.rst of this directory should be written.